epub34/authoring/index.html

```diff
@@ -3109,854 +3108,921 @@ <h3>Parsing URLs in the package document</h3>
 					document as <var>base</var>.</p>
 			</section>
 
-			<section id="sec-shared-attrs">
-				<h3>Shared attributes</h3>
-
-				<p>This section provides definitions for shared attributes (i.e., attributes allowed on two or more
-					elements).</p>
+			<section id="sec-vocab-assoc">
+				<h3>Vocabulary association mechanisms</h3>
 
-				<section id="attrdef-dir" data-epubcheck="true"
-					data-tests="https://w3c.github.io/epub-structural-tests/#05-package-document_package-document.feature_L24">
-					<h4>The <code>dir</code> attribute</h4>
+				<section id="vocab-assoc-intro">
+					<h3>Introduction</h3>
 
-					<div class="note">
-						<p>The <code>dir</code> attribute currently does not have sufficient <a
-								href="https://www.w3.org/policies/process/#adequate-implementation">implementation
-								experience</a>, as [=reading systems=] typically only support a single default
-							directionality for metadata display.</p>
-						<p>Although it lacks the necessary reading system support, it is integral to the content model
-							on which EPUB is built (i.e., for internationalization support in the [=package document=]).
-							Consequently, it is strongly encouraged to set the proper directionality of text values in
-							the package document to ensure proper rendering once this situation improves.</p>
-						<p>The lessening of requirements for this attribute was only done to account for the different
-							process under which EPUB was developed prior to being brought into W3C. New features
-							developed under W3C processes will not receive a similar exemption.</p>
-					</div>
+					<p>EPUB defines a method of referencing terms and properties defined in metadata and semantic
+						vocabularies using the <a href="#sec-property-datatype"><var>property</var> data type</a>. The
+							<code>property</code> and <code>rel</code> attributes use the data type, for example, to
+						define properties and relationships in the [=package document=].</p>
```

Comment:
Just wondering... I would think that adding `epub:type` directly into this list (with a parenthetical note saying that `epub:type` is not used in the package document) is better than having a separate note at the end of the section. As it is now, it suggests that `epub:type` usage is, sort of, an afterthought or secondary feature (which is not).